README.md

```diff
@@ -0,0 +1,55 @@
+# What is Weechat-Matrix?
+
+[Weechat](https://weechat.org/) is an extensible chat client.
+
+[Matrix](https://matrix.org/blog/home) is an open network for secure, decentralized communication.
+
+[Weechat-Matrix](https://github.com/poljar/weechat-matrix/) is a Python plugin for Weechat that lets Weechat communicate over the Matrix protocol.
+
+# Project Status
+
+Weechat-Matrix already supports large parts of the Matrix protocol.  See our [TODO](https://github.com/poljar/weechat-matrix/blob/master/TODO.md) list for more details.  Encryption support is still in the works.
+
+# Installation
+
+Installation is easy.  As your regular user, just run: `make install` in this repository directory.
+
+The following python modules must also be available on your system:
+
+- webcolors
+- http\_parser
```

Comment:
I think that there are more dependencies, but we can add them later on as we remember them.

Reply:
I will try it on a clean install and check